src/test/regress/sql/gp_query_id.sql

```diff
@@ -0,0 +1,148 @@
+--
+-- Test the query command identification
+--
+create extension if not exists gp_inject_fault;
```

Comment:
When the test failed, I fix it and run it again, the test fail here, because client_min_messages = notice. Can we place ```create extension ``` into ```start_ignore``` or set client_min_messages at the beginning of the file?

Reply:
moved `create extension` into the ignore section